src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -970,7 +976,7 @@ def inventory_schema(make_schema):
 @pytest.fixture
 def make_catalog(ws, sql_backend, make_random) -> Generator[Callable[..., CatalogInfo], None, None]:
     def create() -> CatalogInfo:
-        name = f"ucx_C{make_random(4)}".lower()
+        name = f"ucx_c{make_random(4).lower()}_{get_purge_suffix()}"
```

Comment:
done via properties. purge suffix is irrelevant here

Reply:
I've updated this to set a `RemoveAfter` tag on the catalog instead.

Reply:
As noted elsewhere, I've dropped the tags as well and added a note in the code that catalogs can't currently be protected from purging by the watchdog.